doc/concepts/interconnectivity.rst

```diff
@@ -39,6 +39,17 @@ A recipe lets you define which sites are connected to which.
    action potential
       Spikes travel over :term:`connections <connection>`. In a synapse, they generate an event.
 
+   spike detector
```

Comment:
Have a `:ref:` to Python's `arbor.spike_detector`. It may be confusing to define the concept without referencing the class.

Reply:
Also, there is a stub in `/doc/concepts/decor.rst` for this. Maybe move this documentation there, since talking specifically about being placed (should that list of placeables be termified then?). A second (more abstract?) definition of a `spike detector` might be confusing.

Reply:
It's now cross-referenced and cross-explained in simulation and interconnectivity, plus API and placing ... enough.